### DISCUSSION TITLE
Local password on Element Desktop

### DISCUSSION THREAD

**Discussion by eNkKVS4HxHc9jaq** [element-hq/element-meta#1690] [Ideas]
This functionality already exists in the mobile version. I myself everyday have to live with the fact that anybody who can locally access my machine can just launch Element and read any of my messages. 

Sure, you can log out, but then you also have to log back in. This takes ages, day by day, and you eventually end up with just leaving it unlocked. My account password is a long one which no human could remember, and I like it that way. For local lock, I would use a simpler password which I can remember. Yet still, it would vastly improve the security if the app would just lock automatically by a mere simple password when I'm away for a set amount of time.

This is honestly one of the worst aspects of Element security wise in my opinion. Even Telegram has this, even though it is not really privacy based.

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/3777
